yt/frontends/ramses/field_handlers.py

```diff
@@ -324,6 +327,58 @@ def detect_fields(cls, ds):
 
         return fields
 
+class GravFieldFileHandler(FieldFileHandler):
+    ftype = 'gravity'
+    fname = 'grav_{iout:05d}.out{icpu:05d}'
+    config_field = 'ramses-grav'
+
+    attrs = ( ('ncpu', 1, 'i'),
+              ('nvar', 1, 'i'),
+              ('nlevelmax', 1, 'i'),
+              ('nboundary', 1, 'i')
+    )
+
+    @classmethod
+    def any_exist(cls, ds):
+        files = os.path.join(
+            os.path.split(ds.parameter_filename)[0],
+            'grav_?????.out00001')
+        ret = len(glob.glob(files)) == 1
+
+        cls._any_exist = ret
```

Comment:
storing this kind of state on the class object itself is a bad idea IMO (what happens when someone simultaneously loads two different RAMSES outputs in the same python session?

Better to make this method and the one below regular methods and store the state on the instance, not the class (e.g. `self` instead of `cls`).

Reply:
I'm removing them as they're not used anywhere and are potentially bogus -- as you pointed out.